src/main/java/org/reactome/release/qa/common/QACheckerHelper.java

```diff
@@ -445,6 +452,27 @@ private static boolean hasSpeciesAttribute(GKInstance physicalEntity) {
         return physicalEntity.getSchemClass().isValidAttribute(ReactomeJavaConstants.species);
     }
 
+    /**
+     * Helper method that finds all unique DbIds in the 'species' and 'relatedSpecies' attributes of the instance.
+     * @param inst - GKInstance, an Event or PhysicalEntity.
+     * @return - Set<Long>, DbIds from species/relatedSpecies attributes.
+     * @throws Exception, thrown by MySQLAdaptor.
```

Comment:
Similar to https://github.com/reactome/release-qa/pull/26/files#r544613631.  Could you mention the exception is thrown when an exception is thrown getting the inst species or relatedSpecies.